doc/xroad_catalog_installation_guide.md

```diff
@@ -203,14 +203,14 @@ in effect:
 | `xroad-catalog.country.fi.fetch.external-limit`             | 500      | A parameter for setting the maximum amount of Members that should be fetched per external API in one run, e.g. value `500` means `500 members`. In the current implementation the example value would fetch `500` members information from both the `company` and `organization` API.                                            |
 | `xroad-catalog.country.fi.fetch.external-update-after-days` | 7        | A parameter for setting the amount of days after which the X-Road Catalog Collector should consider Company and Organization data stale and try to fetch data from the external API again, e.g. value `7` means `after 7 days`.                                                                                                  |
 
-In addition, update the `xroad-catalog.shared-params-file` property value in `/etc/xroad/xroad-catalog/lister-production.properties`.
+In addition, update the `xroad-catalog.shared-params-file` property value.
```

Comment:
I feel we should still leave a note here stating that this is for the lister configuration (as the previous ones are for the collector). It might in fact make sense to split it with a header. And then the passwords part keep as a common section below.